src/configs/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const Constants = {
```

Comment:
I would not store every constant inside an object, unless they belong together and are passed around all at once. Also a dedicated, global constant should always follow the *SCREAMING_SNAKE_CASE* naming convention.